common/appointment/create.ts

```diff
@@ -55,8 +56,13 @@ export const createMatchAppointments = async (matchId: number, appointmentsToBeC
     const pupilUserId = getUserIdTypeORM(pupil);
     const hosts = [student];
 
-    const videoChat = await createZoomMeetingForAppointments(hosts, appointmentsToBeCreated, false);
-    logger.info(`Zoom - Created meeting ${videoChat.id} for match ${matchId} with ${appointmentsToBeCreated.length} appointments`);
+    let zoomMeetingId: string | null;
+
+    if (isZoomFeatureActive()) {
```

Comment:
previously this would silently do nothing somewhere nested, and return some fake data, I'd prefer to explicitly toggle this off here